SCANNER_VERSION

```diff
@@ -1 +1 @@
-2.33.5
```

Comment:
Scanner `2.33.x` aligns to ACS `4.4.x`

Scanner `2.34.x` aligns to ACS `4.5.x`

If there is no scanner `2.33.x` tag that fixes these vulns we should make the changes to the `2.33` branch and create a new `2.33.6` tag to keep ACS `4.4` and scanner `2.33` aligned.

Reply:
I'm working on this now (new scanner release `2.33.6`)

Reply:
`2.33.6` is avail now FYI